modules/engine/src/shader-inputs.ts

```diff
@@ -88,6 +88,10 @@ export class ShaderInputs<
       const moduleName = name as keyof ShaderPropsT;
       const moduleProps = props[moduleName];
       const module = this.modules[moduleName];
+      if (!module) {
+        // Ignore props for unregistered modules
```

Comment:
Hmmm. .. I kind of feel that we should `log.warn(...)()` here? 

I get that the proposed use case is that deck wants to do "sloppy" prop settings on all models (in which case even a deduplicated warning would not be desirable), but at the same time, if we don't report this, it could be a very annoying facepalm for users that we silently accept module settings that aren't supported.

Perhaps we could provide a way to check if a module is supported?

Reply:
Yes I considered that, but having that warning permanently in deck would be very annoying. That said, I'll add the warning for now and will try and integrate it in deck using `Object.keys(shaderInputs.modules)` to check for allowed modules